x/programs/rust/sdk-macros/src/lib.rs

```diff
@@ -157,6 +157,9 @@ pub fn public(_: TokenStream, item: TokenStream) -> TokenStream {
 
             #[no_mangle]
             unsafe extern "C" fn #name(args: wasmlanche_sdk::HostPtr) {
+                #[cfg(debug_assertions)]
+                wasmlanche_sdk::panic::register_panic();
```

Comment:
It would be better to hide this such that `register_panic` is just a no-op for `cfg(not(debug_assertions))`

Reply:
Good idea, it's less error-prone